_posts/minutes/2016-03-18-minutes.markdown

```diff
@@ -1 +1,35 @@
+---
+---
+Begin: 01:05pm 
+### Introduction
+##A-Reports: 
+  -ICC: Spring Events:  
+  -Event Committee Report: Shirt Counts, Field Trip Sign-Up, Garden Fest & Extra Life Volunteer Sign-UpExtra Life Theme:         Shooter Games (Halo, CoD, GoW); Racing Games Tournament Room (Super Smash Bros) -Add on Info. For the events (Extra Life)   -Garden Fest: 9am-4pm 
+  -Treasurer Report (N/A) 
+  
+##B-Nomination 
+  (1:23pm): New Board Members are serve from Fall 2016 through Summer 2017 Old Minutes Review: 
+    -Nominations for Secretary: Oscar, Diego 
+    -Nominations for President: Vananh Vo 
+    -Nominations for Treasurer: Oscar, Edgar 
+  -Nominations for ICC: Diego, Alec 
+### Counts for Official Members:  
+  Votes: 
+  President: Vananh      
+  Secretary: Diego                   
+  Treasurer: Edgar                     
```

Comment:
Only need to include two spaces at the end of the line, not ~20 :stuck_out_tongue: